packages/api/test/nfts-upload.spec.js

```diff
@@ -330,92 +330,91 @@ test.serial('should upload a single CAR file', async (t) => {
   t.is(data.content.dag_size, 15, 'correct dag size')
 })
 
-test.serial(
-  'should check dag completness with linkdex-api for partial CAR',
-  async (t) => {
-    const client = await createClientWithUser(t)
-    const config = getTestServiceConfig(t)
-    const mf = getMiniflareContext(t)
+// TODO verify with @alanshaw that we don't need to do this in the new upload flow
```

Comment:
of course this means we should remove it also from the code
https://github.com/nftstorage/nft.storage/blob/ad33faf83ae1c994f040f6ad1539496b16ad264a/packages/api/src/routes/nfts-upload.js#L190-L206
https://github.com/nftstorage/nft.storage/blob/ad33faf83ae1c994f040f6ad1539496b16ad264a/packages/api/src/routes/nfts-upload.js#L220-L225

Reply:
yep! I'm planning on removing the whole legacy codepath plus these tests once this is running reliably in production